docs/admin/reference/hardware.rst

```diff
@@ -52,118 +60,56 @@ The 2nd-generation ThinkPad T14 **with an 11th-generation Intel Core processor**
-
-        sudo chown amnesia:amnesia ~amnesia/Desktop/<fileName.iso>
-
-- Verify the checksum of the downloaded ISO file using the following command, comparing it against the checksum in the file listing above:
-
-  .. code-block:: sh
-
-    sha256sum /path/to/downloaded.iso
-
-- Create a USB-bootable version of the ISO using the command:
-
-  .. code-block:: sh
-
-    geteltorito <path/to/CDISO> > usb-bios.iso
-
-  .. note:: To install the ``geleltorito`` utility on Debian-based systems, use the command
-
-    .. code-block:: sh
-
-      sudo apt install genisoimage
-
-    To install it on Fedora-based systems, use the command:
-
-    .. code-block:: sh
-
-      sudo dnf install geteltorito genisoimage
-
-- Plug in a USB and check its device name with the ``lsblk`` command - use the root device name below, not a partition (eg. ``/dev/sdc`` instead of ``/dev/sdc1``).
-
-- Write the BIOS update ISO to the USB using the following command:
-
-  .. code-block:: sh
-
-    sudo dd if=usb-bios.iso of=/dev/sdX bs=1M && sync
-
-  where ``sdX`` is the device name verified above.
-
-  .. caution::
+The Qubes Hardware Compatibility List (HCL)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-    The ``dd`` command will wipe data on the targeted device. Make sure that you use the correct device name.
+The `Qubes Hardware Compatibility List (HCL) <https://www.qubes-os.org/hcl/>`_
+is a community-maintained list of hardware that has been tested by Qubes users.
+It consists of individual reports generated and submitted by Qubes users across
+the world. Anyone can attempt to install Qubes on their computer, then report
+back on whether or not it can be installed, if there are any issues, and overall,
+what the experience is like.
 
-  Once complete, remove the USB.
+There are some benefits to this list:
 
-- Plug the USB into the ThinkPad.
+* A much wider selection of hardware is tested, because anyone can contribute to the list
+* There are sometimes multiple reports for a particular system, which lets you compare and feel confident the results are consistent
+* It tells you exactly what is and isn't working within the system, so you can decide if a device you own will function well enough to suit your needs
+* Devices get tested across many different configurations and Qubes versions
 
-- Boot the ThinkPad and follow the prompts to enter its startup and boot menus, likely via the <Enter> and <F12> keys, respectively.
+However, there are some things to consider:
 
-- Follow the on-screen instructions to update the BIOS, including any mandatory reboots. Note that the instructions may refer to an update CD instead of your update USB.
+* Reports are not verified for their accuracy by either the Qubes team or Freedom of the Press Foundation
+* Reports don't automatically get tested with newer Qubes releases
+* Reports may reflect the experience on older Qubes versions, rather than the current version (where hardware support may have improved)
```

Comment:
Suggest refactoring thse two bullet points:

* Reports correspond to a specific Qubes OS version, and may not reflect breaking changes or expanded  hardware support in the most recent Qubes OS version